modules/registry/go.mod

```
@@ -0,0 +1,53 @@
+module github.com/testcontainers/testcontainers-go/modules/registry
+
+go 1.19
+
+require (
+	github.com/testcontainers/testcontainers-go v0.20.0
+	gotest.tools/gotestsum v1.10.0
+)
+
+require (
+	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
+	github.com/Microsoft/go-winio v0.5.2 // indirect
+	github.com/cenkalti/backoff/v4 v4.2.0 // indirect
+	github.com/containerd/containerd v1.6.19 // indirect
+	github.com/cpuguy83/dockercfg v0.3.1 // indirect
+	github.com/dnephin/pflag v1.0.7 // indirect
+	github.com/docker/distribution v2.8.1+incompatible // indirect
```

Let's add this security fix too, that was merged today in #1189 
```suggestion
	github.com/docker/distribution v2.8.2+incompatible // indirect
```